serde_with/src/de/impls.rs

```diff
@@ -1,4 +1,6 @@
 use crate::{formats::*, prelude::*};
+#[cfg(feature = "hashbrown")]
+use hashbrown::{HashMap as HashbrownMap, HashSet as HashbrownSet};
```

Comment:
Maybe it is better to add the version number into the renames here too, similar to the `indexmap_2` case. You would need to propagate that to the `de` and `ser` modules.

```suggestion
use hashbrown::{HashMap as HashbrownMap014, HashSet as HashbrownSet014};
```